distribution/lib/Standard/Microsoft/0.0.0-dev/src/Internal/SQLServer_Connection.enso

```diff
@@ -0,0 +1,299 @@
+private
+
+from Standard.Base import all
+import Standard.Base.Metadata.Display
+from Standard.Base.Metadata.Choice import Option
+from Standard.Base.Metadata.Widget import Single_Choice
+
+import Standard.Table.Rows_To_Read.Rows_To_Read
+from Standard.Table import Table
+
+import Standard.Database.Column_Description.Column_Description
+import Standard.Database.Connection.Connection.Connection
+import Standard.Database.DB_Table.DB_Table
+import Standard.Database.Internal.Connection.Entity_Naming_Properties.Entity_Naming_Properties
+import Standard.Database.Internal.JDBC_Connection
+import Standard.Database.SQL_Query.SQL_Query
+import Standard.Database.SQL_Statement.SQL_Statement
+from Standard.Database.Connection.Connection import make_database_selector, make_schema_selector, make_structure_creator, make_table_name_selector, make_table_types_selector
+from Standard.Database.Errors import SQL_Error, Table_Already_Exists, Table_Not_Found
+from Standard.Database.Internal.Upload_Table import first_column_name_in_structure
+
+import project.Internal.SQLServer_Dialect
+
+type SQLServer_Connection
+    ## PRIVATE
+
+       Creates a SQLServer connection based on a URL and properties.
+
+       Arguments:
+       - url: The URL to connect to.
+       - properties: A vector of properties for the connection.
+       - make_new: A function that returns a new connection.
+    create : Text -> Vector -> (Text -> Text -> SQLServer_Connection) -> SQLServer_Connection
+    create url properties make_new =
+        jdbc_connection = JDBC_Connection.create url properties
+        entity_naming_properties = Entity_Naming_Properties.from_jdbc_connection jdbc_connection is_case_sensitive=False
```

Comment:
So T-SQL is truly case insensitive, interesting.

It would be nice to create a test in which we do `(Table.new [["A", [1]], ["a", [2]]]).select_into_database_table ...` and see what happens.

Reply:
Of course we can add it in next iterations, just noting so that we can keep track of that.

Reply:
https://github.com/enso-org/enso/issues/10710